plugins/controlboard/include/ControlBoardData.hh

```diff
@@ -66,6 +67,8 @@ struct ActuatedAxisProperties
 {
     CommonJointProperties commonJointProperties;
     std::unique_ptr<yarp::dev::gzyarp::TrajectoryGenerator> trajectoryGenerator;
+    std::unique_ptr<yarp::dev::gzyarp::RampFilter> speed_ramp_handler;
+    std::unique_ptr<yarp::dev::gzyarp::Watchdog> velocity_watchdog;
```

Comment:
Can you use `camelCase` for consistency with the rest of the attributes?

Reply:
Style fixed